tests/conftest.py

```diff
@@ -199,7 +198,6 @@
         CUSTOM_TASK_INTERFACE_PYTORCH_BINARY,
         CUSTOM_TASK_INTERFACE_PYTORCH_MULTICLASS,
     ],
-    PYTHON311_GENAI: [PYTHON_TEXT_GENERATION, PYTHON_AGENTIC_WORKFLOW],
```

Comment:
Removing these was ok as these test case was never triggered in this env as you removed it from test_per_framewor.json